understanding/20/resize-text.html

```diff
@@ -59,7 +59,7 @@ <h2>Intent of Resize text</h2>
       </p>
       
       <p>Content satisfies the Success Criterion if it can be scaled up to 200%, that is, up
-         to twice the width and height. Authors may support scaling beyond that limit, however,
+         to twice the width and height. Content that can be scaled up to 200% using user agent zoom meets this Success Criteria, even if the text alone cannot be scaled up to 200%.  Authors may support scaling beyond that limit, however,
```

Comment:
I am not sure is "even if the text alone cannot be scaled up to 200%." will be understood correctly. What is meant is "even if the text only-scaling function of a user agent alone cannot scale up text to 200%".
The way it is written may leave people to think that if they use browser zoom and the text is simulataneously scaled down via CSS at narrower breakpoints as they zoom, this would meet 1.4.4